node_modules/postcss-initial/~/.config/configstore/update-notifier-npm.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What is this? Since it is part of node modules do we this needed?

Reply:
Yeah I don't think we need it. I have unstaged it.